frontend/src/lib/util.ts

```diff
@@ -394,6 +395,16 @@ export function useId(prefix = '') {
   return id;
 }
 
+export function dispatchHeadlampEvent(eventInfo: HeadlampEvent | HeadlampEventType) {
```

Comment:
I don't think this should be in util. Because it's a grab bag of different things and it makes things complicated with import circles.

Reply:
I am happy to move it to a more appropriate place. What do you suggest?